Rakefile

```diff
@@ -0,0 +1,158 @@
+    Dir.chdir("#{repository}") do
+      sh "git submodule update --init --recursive"
+    end
+  end
+end
+
+## [ Release a new version ] ##################################################
+
+namespace :release do
+  desc 'Create a new release on GitHub, CocoaPods and Homebrew'
+  task :new => [:check_versions, 'swiftgen:tests', :github, :cocoapods, :homebrew]
+
+  desc 'Check if all versions from the podspecs and CHANGELOG match'
+  task :check_versions do
+    results = []
+
+    # Check if bundler is installed first, as we'll need it for the cocoapods task (and we prefer to fail early)
+    `which bundler`
+    results << Utils.log_result( $?.success?, 'Bundler installed', 'Please install bundler using `gem install bundler` and run `bundle install` first.')
+
+    # Extract version from SwiftGen.podspec
+    version = Utils.podspec_version
+    Utils.log_info('SwiftGen.podspec', version)
+
+    # Check if version matches the Info.plist
+    results << Utils.log_result(version == Utils.plist_version, "Info.plist version matches", "Please update the version numbers in the Info.plist file")
+
+    # Check if entry present in CHANGELOG
+    changelog_entry = system(%Q{grep -q '^## #{Regexp.quote(version)}$' SwiftGen/CHANGELOG.md})
+    results << Utils.log_result(changelog_entry, "CHANGELOG, Entry added", "Please add an entry for #{version} in CHANGELOG.md")
+
+    changelog_master = system(%q{grep -qi '^## Master' SwiftGen/CHANGELOG.md})
+    results << Utils.log_result(!changelog_master, "CHANGELOG, No master", 'Please remove entry for master in CHANGELOG')
+
+    exit 1 unless results.all?
+
+    print "Release version #{version} [Y/n]? "
+    exit 2 unless (STDIN.gets.chomp == 'Y')
+  end
+
+  desc 'Create a zip containing all the prebuilt binaries'
+  task :zip => ['swiftgen:clean', 'swiftgen:install'] do
+    `cp SwiftGen/LICENSE SwiftGen/README.md SwiftGen/CHANGELOG.md SwiftGen/build/swiftgen`
+    `cd SwiftGen/build/swiftgen; zip -r ../swiftgen-#{Utils.podspec_version}.zip .`
+  end
+
+  def post(url, content_type)
+    uri = URI.parse(url)
+    req = Net::HTTP::Post.new(uri, initheader = {'Content-Type' => content_type})
+    yield req if block_given?
+    req.basic_auth 'AliSoftware', File.read('.apitoken').chomp
+
+    response = Net::HTTP.start(uri.host, uri.port, :use_ssl => (uri.scheme == 'https')) do |http|
+      http.request(req)
+    end
+    unless response.code == '201'
+      puts "Error: #{response.code} - #{response.message}"
+      puts response.body
+      exit 3
+    end
+    JSON.parse(response.body)
+  end
+
+  desc 'Upload the zipped binaries to a new GitHub release'
+  task :github => :zip do
+    v = Utils.podspec_version
+
+    changelog = `sed -n /'^## #{v}$'/,/'^## '/p CHANGELOG.md`.gsub(/^## .*$/,'').strip
+    print_info "Releasing version #{v} on GitHub"
+    puts changelog
+
+    json = post('https://api.github.com/repos/SwiftGen/SwiftGen/releases', 'application/json') do |req|
```

Comment:
Yeah good question, but I think if you have push access to the repo as you have you might be able to use the API with your same GitHub token too and create releases with it too, just like I think you can already do manually?